src/virtualenv/run/__init__.py

```diff
@@ -88,32 +89,32 @@ def handle_extra_commands(options):
 
 
 def load_app_data(args, parser, options):
+    parser.add_argument(
+        "--readonly-app-data",
+        action="store_true",
+        help="allow reading from the configured app data even if the app data directory is readonly",
```

Comment:
We're not actually checking the app data is read-only, and we really don't care if it's or not. We just enforce it via logic that we don't update it, so a better message here is `use app data folder in read-only mode (write operations will fail with error)`